src/layout/IHyprLayout.hpp

```diff
@@ -155,9 +155,23 @@ class IHyprLayout {
     */
     virtual void replaceWindowDataWith(CWindow* from, CWindow* to) = 0;
 
+    /*
+        Determines if a window can be focused. If hidden this usually means the window is part of a group.
+		*/
```

Comment:
why is the indent here so weird? with the other too

Reply:
what do you mean?
![image](https://github.com/hyprwm/Hyprland/assets/83010835/9ed2f5b5-f98b-4727-93b0-26dd6b71fb55)

Reply:
It appears clang-format dosen't remove tabs in comments, sorry
![image](https://github.com/hyprwm/Hyprland/assets/83010835/094c66d5-d226-4741-97e6-14896d471e82)